.gitignore

```diff
@@ -0,0 +1,212 @@
+# Created by .ignore support plugin (hsz.mobi)
```

Comment:
Thoughts on renaming this repo to:
> core-case-data-store-client
i.e. CCD has lots of microservices, is the intention that this client should be used for all, or just the data store client?

Reply:
done

Reply:
I'm not happy with this file, I'd prefer a minimal one. I think it's better to consciously add a `.gitignore` entry when there is a need for it because a new tool/library is used. With this file we may hide something we don't want to and it's so hard to troubleshoot when such problem happens.